tests/test_core_to_contrib.py

```diff
@@ -24,1741 +24,8 @@
-    ),
-    (
-        'airflow.providers.apache.hive.transfers.hive_to_mysql.HiveToMySqlOperator',
-        'airflow.operators.hive_to_mysql.HiveToMySqlTransfer',
-    ),
-    (
-        'airflow.providers.apache.hive.transfers.mysql_to_hive.MySqlToHiveOperator',
-        'airflow.operators.mysql_to_hive.MySqlToHiveTransfer',
-    ),
-    (
-        'airflow.providers.apache.hive.transfers.s3_to_hive.S3ToHiveOperator',
-        'airflow.operators.s3_to_hive_operator.S3ToHiveTransfer',
-    ),
-    (
-        'airflow.providers.apache.hive.transfers.hive_to_samba.HiveToSambaOperator',
-        'airflow.operators.hive_to_samba_operator.HiveToSambaOperator',
-    ),
-    (
-        'airflow.providers.apache.hive.transfers.mssql_to_hive.MsSqlToHiveOperator',
-        'airflow.operators.mssql_to_hive.MsSqlToHiveTransfer',
-    ),
-    (
-        'airflow.providers.microsoft.azure.transfers.file_to_wasb.FileToWasbOperator',
-        'airflow.contrib.operators.file_to_wasb.FileToWasbOperator',
-    ),
-    (
-        'airflow.providers.google.suite.transfers.gcs_to_gdrive.GCSToGoogleDriveOperator',
-        'airflow.contrib.operators.gcs_to_gdrive_operator.GCSToGoogleDriveOperator',
-    ),
-    (
-        'airflow.providers.microsoft.azure.transfers.oracle_to_azure_data_lake'
-        '.OracleToAzureDataLakeOperator',
-        'airflow.contrib.operators.oracle_to_azure_data_lake_transfer.OracleToAzureDataLakeOperator',
-    ),
-    (
-        'airflow.providers.oracle.transfers.oracle_to_oracle.OracleToOracleOperator',
-        'airflow.contrib.operators.oracle_to_oracle_transfer.OracleToOracleTransfer',
-    ),
-    (
-        'airflow.providers.google.cloud.transfers.s3_to_gcs.S3ToGCSOperator',
-        'airflow.contrib.operators.s3_to_gcs_operator.S3ToGCSOperator',
-    ),
-    (
-        'airflow.providers.mysql.transfers.vertica_to_mysql.VerticaToMySqlOperator',
-        'airflow.contrib.operators.vertica_to_mysql.VerticaToMySqlTransfer',
-    ),
-    (
-        'airflow.providers.mysql.transfers.presto_to_mysql.PrestoToMySqlOperator',
-        'airflow.operators.presto_to_mysql.PrestoToMySqlTransfer',
-    ),
-    (
-        'airflow.providers.google.cloud.operators.cloud_storage_transfer_service'
-        '.CloudDataTransferServiceS3ToGCSOperator',
-        'airflow.contrib.operators.s3_to_gcs_transfer_operator.CloudDataTransferServiceS3ToGCSOperator'
-    ),
-    (
-        'airflow.providers.google.cloud.transfers.cassandra_to_gcs.CassandraToGCSOperator',
-        'airflow.contrib.operators.cassandra_to_gcs.CassandraToGoogleCloudStorageOperator',
-    ),
-]
-
-ALL = HOOKS + OPERATORS + SECRETS + SENSORS + TRANSFERS
-
-RENAMED_HOOKS = [
-    (old_class, new_class)
-    for old_class, new_class in HOOKS + OPERATORS + SECRETS + SENSORS
-    if old_class.rpartition(".")[2] != new_class.rpartition(".")[2]
-]
+from airflow.models.baseoperator import BaseOperator
+from tests.deprecated_classes import ALL, RENAMED_ALL
```

Comment:
I think we should move the full file. Its name is invalid because it also contains tests for messages that are not part of AIP-21.